modules/mono/editor/bindings_generator.cpp

```diff
@@ -414,20 +457,8 @@ String BindingsGenerator::bbcode_to_xml(const String &p_bbcode, const TypeInterf
 					xml_output.append("</c>");
 				}
 			} else if (link_tag == "constant") {
-				if (!target_itype || !target_itype->is_object_type) {
-					if (OS::get_singleton()->is_stdout_verbose()) {
-						if (target_itype) {
-							OS::get_singleton()->print("Cannot resolve constant reference for non-Godot.Object type in documentation: %s\n", link_target.utf8().get_data());
-						} else {
-							OS::get_singleton()->print("Cannot resolve type from constant reference in documentation: %s\n", link_target.utf8().get_data());
-						}
-					}
-
-					// TODO Map what we can
-					xml_output.append("<c>");
-					xml_output.append(link_target);
-					xml_output.append("</c>");
-				} else if (!target_itype && target_cname == name_cache.type_at_GlobalScope) {
+				if (link_target_parts[0] == name_cache.type_at_GlobalScope) {
+				find_constant_in_global_scope:
```

Comment:
Nitpicking: I would prefer `find_constant_in_global_scope` to be a function.

Reply:
I've extracted the BBCode tags XML generation to their own separate method. It's a separate commit so let me know if you don't like it and I can just drop it or redo it differently.